client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts

```diff
@@ -8,17 +8,34 @@ import {
   ArrayUtils,
   useCurrency,
   useUrlQueryParams,
+  InvoiceLineNodeType,
+  ColumnAlign,
 } from '@openmsupply-client/common';
 import { LocationRowFragment } from '@openmsupply-client/system';
 import { InboundItem } from './../../../types';
 import { InboundLineFragment } from '../../api';
 
+const isDefaultPlaceholderRow = (row: InboundLineFragment) =>
+  row.type === InvoiceLineNodeType.StockIn && !row.numberOfPacks;
```

Comment:
numberOfPacks is non-optional so IMHO more readable to do `row.numberOfPacks === 0`

Reply:
yeah, I agree. thanks - had taken the nullable from the outbound 'unallocated lines' which is a different scenario.